cmd/ostracon/commands/show_validator.go

```diff
@@ -15,18 +18,29 @@ var ShowValidatorCmd = &cobra.Command{
 	Use:     "show-validator",
 	Aliases: []string{"show_validator"},
 	Short:   "Show this node's validator info",
-	RunE:    showValidator,
-	PreRun:  deprecateSnakeCase,
+	RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
Do you use the value specified in config to decide whether to return information on kms node or not?

Reply:
To the first sentence, yes, I do. And what's the relationship between the first and second sentence of your point? Are you implying that this anonymous function should be given an easy-to-understand name? Even if I do that, I don't see how it would improve anything.

Reply:
Sorry for lack of the information. It was not an anonymous function, but the part of the processing after `if config.PrivValidatorListenAddr! =" "{` Inside `showValidator`. Having a function that clearly states that it is a Kms node helps to read the intent.

Reply:
At this point, I don't think the code is so difficult that it's necessary to separate functions with and without `config.PrivValidatorListenAddr` for semantics and ease of testing. We may do so in future revisions if the if-else statements become more complex.

Reply:
It's easy for those who know it, but it's not easy for those who don't.  I approve in the hope that the documentation will be enriched.